config/locales/application_form.yml

```diff
@@ -45,3 +42,18 @@ en:
         label: Year qualification was awarded
       button:
         add_degree: Add another degree
+    qualification_section:
+      heading: Accademic qualifications
+      type:
+        label: Type of qualification
+        hint_label: Add Maths or English GCSE, or other relevant qualifications
```

Comment:
This wording is different to the prototype: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training-prototype/blob/master/app/views/application/qualifications/other.njk#L30